plugins/modules/docker_image.py

```diff
@@ -179,8 +180,10 @@
     default: no
   repository:
     description:
-      - Full path to a repository. Use with state C(present) to tag the image into the repository. Expects
-        format I(repository:tag). If no tag is provided, will use the value of the C(tag) parameter or I(latest).
+      - Use with state C(present) to tag the image.
+      - Expects format C(repository:tag). If no tag is provided, will use the value of the I(tag) parameter or C(latest).
+      - If I(push=true), I(repository) must either include a registry, or will be assumed to belong to the default
```

Comment:
I may misunderstood what you have meant by including a registry though.
```suggestion
      - If I(push=true), I(repository) must either be a registry, or will be assumed to belong to the default
```

Reply:
A registry is "your.registry.com:5000", a name including a registry is "your.registry.com:5000/repo/image". I mean the latter here.

Reply:
Ah, sure, thanks for clarification.